waggle-dance-core/pom.xml

```diff
@@ -285,6 +286,16 @@
         <artifactId>spring-boot-maven-plugin</artifactId>
         <version>${spring-boot-maven-plugin.version}</version>
         <configuration>
+          <requiresUnpack>
+            <!-- We are hitting an issue introduced in hive-common-2.1.1 and fixed in https://issues.apache.org/jira/browse/HIVE-17155
```

Comment:
Would we want to undo this when we move to a newer version of Hive? If so maybe a TODO with > version X remove?